agenta-cli/agenta/cli/main.py

```diff
@@ -171,18 +172,17 @@ def init(app_name: str, backend_host: str):
 
         filtered_org = None
         if where_question == "On agenta cloud":
-            which_organization = questionary.select(
-                "Which organization do you want to create the app for?",
-                choices=[
-                    f"{org.name}: {org.description}" for org in user_organizations
-                ],
-            ).ask()
+            if not organisation:
```

Comment:
I would add 
```
if len(user_organizations)==1:
     organization = user_organizations[0]
else
    the rest
```

Reply:
I don't understand the logic, 
if not organisation means if no `--organisation organisation_name` is passed

Reply:
Basically if the user has only one organization, we should not ask him to select an organization. Irrespective of anything else, no?